### PR TITLE
Default difficulty set to 1

### LINKED ISSUE #139: A block's minimum difficulty should be 1 instead of 0
in Block.check_constraint, [minimum difficulty is 1](https://github.com/nekoyume/nekoyume/blob/master/nekoyume/models.py#L198). but [Block.create](https://github.com/nekoyume/nekoyume/blob/master/nekoyume/models.py#L477-L480) can difficulty set 0.

### PR DESCRIPTION
**Pull request by iambenzo**
Fixes #139

### CODE REVIEW

**Comment by iambenzo** [comment]
I have no idea what that report means... definitely going to research that!

**Comment by iambenzo** [comment]
Sure thing. How should we proceed?

File: nekoyume/models.py [hunk 1]
```diff
@@ -241,14 +241,14 @@ def valid(self) -> bool:
...
+                valid = valid and self.difficulty == max(1, difficulty + 1)
             elif avg_timedelta > MAX_BLOCK_INTERVAL:
-                valid = valid and self.difficulty == max(0, difficulty - 1)
+                valid = valid and self.difficulty == max(1, difficulty - 1)
             else:
                 valid = valid and self.difficulty == difficulty
         else:
             valid = valid and self.prev_hash is None
-            valid = valid and self.difficulty == 0
+            valid = valid and self.difficulty == 1
```
Comment (ipdae) [inline nekoyume/models.py]: As above.

File: nekoyume/models.py [hunk 2]
```diff
@@ -482,7 +482,7 @@ def create(
             #: Genesis block
             block.id = 1
             block.prev_hash = None
-            block.difficulty = 0
+            block.difficulty = 1
```
Comment (ipdae) [inline nekoyume/models.py]: As above.

**Comment by iambenzo** [comment]
I've had another go. Please do review and let me know how it looks :)

File: nekoyume/models.py (hunk 2 above)
Comment (ipdae) [reply nekoyume/models.py]: @iambenzo you missed this change

**Pull request closed without merging**